src/main/resources/i18n/ru/sbtqa/tag/apifactory/stepdefs/GenericStepDefs/en.properties

```diff
@@ -0,0 +1,4 @@
+userSendRequestNoParams=^user sends request (?:for|to|about) \\((.*?)\\)$
+userSendRequestTableParam=^user sends request (?:for|to|about) \\((.*?)\\) with parameters$
```

Comment:
Our users sometime write "user sends request to (add new popular brand) with parameters:" 
And dont understand why it doesnt work. That's why I suggested it